bitnami/thanos/templates/query-frontend/deployment.yaml

```
@@ -21,6 +21,9 @@ spec:
         app.kubernetes.io/component: query-frontend
       {{- if .Values.queryFrontend.podAnnotations }}
```

Does it make sense to change this condition as well?
```suggestion
      {{- if or .Values.queryFrontend.podAnnotations (include "thanos.queryFrontend.createConfigmap" .) }}
```

---

Nice catch. I pushed a new commit.